lib/transformers/form.js

```diff
@@ -133,20 +122,13 @@ function form({schemaKey, schema, options, transform, data, schemaErrors, fullUi
                         return transformation.content;
                     }
 
-                    // if there is no pageTitle yet, then the schema.title doesn't exist.
-                    // you the question title of the first item in the output order instead.
-                    if (!pageTitle) {
-                        // eslint-disable-next-line prefer-destructuring
-                        pageTitle = transformation.pageTitle;
-                    }
-
                     return nunjucksHelper.toNunjucks(
                         transformation.componentName,
                         transformation.macroOptions
                     );
                 })
                 .join('\n')}`,
-        pageTitle: `${pageTitle} - claim criminal injuries compensation - GOV.UK`
+        pageTitle: summaryText ? `{{ titleErrorPrefix }}${pageHeading}` : `${pageHeading}`
```

Comment:
The second option can be just the pageHeading variable

Reply:
Just had a thought... do you think we should just return some kind of hasErrors/valid flag and handle the prefixing responsibility in CW? e.g.

```
{
    pageTitle,
    content,
    hasErrors: true
}
```

On the CW side it's not so clear what's going on. There's a `titleErrorPrefix` nunjucks variable but nothing referencing it (as it's in the returned title). I reckon this would be a better approach

Reply:
yeah, that makes sense. because then CW has full control over the presentation